src/DwellingImporter.php

```diff
@@ -30,62 +33,76 @@ private function deleteTables()
         $query->execute();
     }
     /**
-     * inserts data from dwellings from API into the db
+     * Inserts the dwellings returned by the api into the database
+     *
+     * @return void
      */
-    private function insertDwellings()
+    private function insertDwellings(): void
```

Comment:
Make sure the whitespace around return type hints is consistent. Looks like () : type is used most commonly